core/services/relay/relay.go

```diff
@@ -12,3 +14,17 @@ var (
 		Terra:  {},
 	}
 )
+
+type VRFRelayer interface {
+	relaytypes.Relayer
+	NewDKGProvider(rargs relaytypes.RelayArgs, transmitterID string) (DKGProvider, error)
+	NewVRFProvider(rargs relaytypes.RelayArgs, transmitterID string) (VRFProvider, error)
+}
+
+type DKGProvider interface {
+	relaytypes.Plugin
+}
+
+type VRFProvider interface {
+	relaytypes.Plugin
+}
```

Comment:
I think it might be more prudent to add these in chainlink-relay https://github.com/smartcontractkit/chainlink-relay/blob/main/pkg/types/types.go